ipfs-7-to-8/migration/config_conv.go

```diff
@@ -0,0 +1,177 @@
+package mg7
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/ipfs/go-cid"
```

Comment:
I used a tool to generate the vendor directory, but it ends up being very big because of all the crypto dependencies. I only use go-cid to verify the peer ID, so alternatively I could copy some of the go-cid code that does verification. @Stebalien which do you think is the better approach?

Reply:
Hm. Actually, looking at that code, we shouldn't be using CID. _Technically_, peer IDs can be decoded as CIDs. However, they're quite the same thing.

I'd actually just remove that code. We should just check "does the address match `/(p2p|ipfs)/QmTargetPeerID$`?". Then we can skip all deps.